packages/teams-js/src/public/app.ts

```diff
@@ -937,7 +942,7 @@ function transformLegacyContextToAppContext(legacyContext: LegacyContext): app.C
       sourceOrigin: legacyContext.sourceOrigin,
     },
     user: {
-      id: legacyContext.userObjectId,
+      id: legacyContext.userObjectId ?? '',
```

Comment:
>userObjectId [](http://example.com/codeflow?start=24&length=12)

`userObjectId` is declared as possibly being `undefined` but `id` cannot be `undefined` so this unexpected case is now translated into the closest corresponding legal value for this property (empty string)